API_CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Breaking changes to the nupic API
+# Breaking changes to the htm API
```

Comment:
Nitpick: I think that this should refer to the old nupic, since these are changes to the API set forth by the old nupic.